docs/animations/GFP_MN.qmd

```diff
@@ -0,0 +1,106 @@
+## GFP Code From Prettymol Ported to MN
+
+Note: WIP. Still some issues to work out
+
+
+```python
```

Comment:
What is happening with this document? Is it intended to be part of the documentation?

Reply:
removed the QMD it was useful for testing but I wouldn't render an entire animation in the docs.  If you want, you can assign me a separate issue to update some of the scripting docs.